app/jobs/google/search_keyword_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchKeywordJob < ApplicationJob
+    def perform(keyword_id)
+      keyword = Keyword.find(keyword_id)
+      search_result = Google::SearchService.new(keyword.name).call
+      update_keyword(keyword, search_result)
+    end
+
+    private
+
+    def update_keyword(keyword, search_result)
+      if search_result
+        keyword.update search_result.merge(status: :parsed)
+      else
+        keyword.update({ status: :failed })
+      end
+    end
```

Comment:
i think we can improve it a bit to reduce the duplication keyword.update

```suggestion
    def update_keyword(keyword, search_result)
      status = search_result ? :parsed : :failed
      keyword.update(status: status, **search_result)
    end
```